src/jit/emitxarch.cpp

```diff
@@ -8368,6 +8373,10 @@ void emitter::emitDispIns(
             {
                 printf("%s, %s", emitRegName(id->idReg1(), EA_PTRSIZE), sstr);
             }
+            else if ((ins == INS_crc32) && (attr != EA_8BYTE))
```

Comment:
@fiigii, might be able to comment better. I copied this from the existing code print code for the R_R form.

I'll go look at the spec and try to determine why this was done in the meantime.

Reply:
@CarolEidt, it looks like this was done because the target is always R32, but the size of the source operand can vary (R8, R16, or R32).

So this ensures that we print `crc32 eax, bx` instead of `crc32 bx, bx` (for example).

We don't currently expose the `Crc32(long, byte)` encoding on the instruction, so we don't need to worry about that case (we only need to worry about `Crc32(long, long)`, where both the target and source are the same size).

Reply:
Fixed with comment.

Reply:
Thanks for checking into that!